app/test_with_resources.bzl

```diff
@@ -1,20 +1,22 @@
-def test_with_resources(file):
-    """
-    Genrule for test files.
+"""
+Macros for app module resources tests.
+"""
+
+def test_with_resources(name):
```

Comment:
Prefer: s/name/file_path/

Reply:
how name before file_path? 

`test_with_resources("src/test/java/org/oppia/android/app/player/state/StateFragmentLocalTest.kt")`

Reply:
Sorry, not sure that I understand. I'm suggesting we use 'file_path' instead of 'name' for this parameter.

Reply:
it gives the error

```
app/test_with_resources.bzl:5: unnamed-macro: By convention "test_with_resources" should have a keyword argument called "name".

It is considered a macro because it calls a rule or another macro "native.genrule" on line 11. (https://github.com/bazelbuild/buildtools/blob/master/WARNINGS.md#unnamed-macro)

```

Reply:
to use `file_path` we need to do as per the official doc below. 

```
If this function is a helper function that's not supposed to be used outside of its file, please make it private (rename it so that the name starts with _), this will prevent loading the function from BUILD files and suppress the warning.
```

Reply:
Ah odd. That's a strange requirement. I guess we need to keep it name then.